contract/hardhat.config.ts

```diff
@@ -0,0 +1,65 @@
+import { HardhatUserConfig, task } from 'hardhat/config'
+import { HttpNetworkUserConfig } from 'hardhat/types'
+import '@nomiclabs/hardhat-etherscan'
+import '@nomiclabs/hardhat-waffle'
+import '@typechain/hardhat'
+import 'hardhat-gas-reporter'
+import 'solidity-coverage'
+import * as dotenv from 'dotenv'
+import * as process from 'process'
+
+dotenv.config()
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task('accounts', 'Prints the list of accounts', async (taskArgs, hre) => {
+    const accounts = await hre.ethers.getSigners()
+
+    for (const account of accounts) {
+        console.log(account.address)
+    }
+})
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+const config: HardhatUserConfig = (() => {
+    function networkConfiguration(
+        url: string | undefined,
+        account: string | undefined
+    ): HttpNetworkUserConfig {
+        return {
+            url: url ?? '',
+            accounts: account ? [`0x${account}`] : 'remote',
+        }
+    }
+
+    return {
+        solidity: '0.8.9',
+        defaultNetwork: 'localhost',
```

Comment:
In my testing PR, I changed this to `hardhat` so `npm test` runs without needing a test node running. Is it safe to change to `hardhat` as the default?

Reply:
Or we can change `package.json` so the tests using `hardhat` network like this:

```
"test": "yarn prettier && hardhat test --network hardhat",
```

Do you have a preference?

Reply:
If no preference, I'll go with the update to `package.json` and will copy over your new hardhat config into my test branch...

Reply:
This is complete and updated in PR #58

Reply:
@briangershon Sounds like a good call. Not sure what the exact difference is between the two, so probably best this way 👍

Reply:
My understanding is that `hardhat` basically runs the blockchain in-process so no need for blockchain node to be running separately. `localhost` expects a node to be running in a separate process/terminal and if not, the tests errors out.